mithril/__init__.py

```diff
@@ -1,6 +1,6 @@
 # (c) 2012 Urban Airship and Contributors
 
-__version__ = (0, 1, 0)
+__version__ = (0, 2, 0)
```

Comment:
This looks like it's technically backwards incompatible (changing from south to django's migrations, dropping python 2.6 support); assuming anybody's using this outside, we should probably try to be good citizens and do a major version bump, no?